### DISCUSSION TITLE
SVG not rendering on Google/Android TV APK build

### DISCUSSION THREAD

**Discussion by pmp02** [zathras/jovial_svg#105] [Q&A] [answered]
Hi

I have a Flutter app which loads and displays some images from a central location.
The application runs on web but on TVs as well.

Recently we want to add support for SVG images but we have some issues with rendering on the TVs.

This is the piece of code that should display the SVG:

```dart
ScalableImageWidget.fromSISource(
              si: ScalableImageSource.fromSvgHttpUrl(
                  Uri.parse(graphicalContentNetwork)));
```

Version: `jovial_svg: ^1.1.21`

The web build works fine (the SVG is supported and renders fine in the web build): `flutter build web --release`
The problem is with the APK build: `flutter build apk --release` . On this build, the SVG is not rendered at all.

Tried also different optimized SVGs (like without STYLE). the issue seems the same (works on web, not on TV). 

Any suggestions are welcomed?

Thanks in advance!

**Comment by zathras** [comment, marked as answer]
Well, what have you tried to debug it?  For example, have you made sure your app has permission to access the network?  How did you test that?  What error messages, if any, show up?  Have you tried a hard-coded asset, like in a string constant?

I'm moving this to a discussion.  Substantially more information about the steps you've taken to identify the real problem would be required before this could be considered an issue with the library.

**Reply by pmp02** [reply]
Hi @zathras,

Thanks for the quick reply. Sorry, I was away few weeks and couldn't follow up on your response.
I haven't got the chance to actually debug on the Android TV and see if there are any errors, but will do that and come back to you.

For sure there are no permissions/network issues as there are more images displayed in a carousel, from the same source and only SVGs are not rendered correctly on the TV (only in the web browser).

Thanks!

**Reply by pmp02** [reply]
Hi, @zathras  
The problem was solved. Indeed, it wasn't an issue with the library, it was an issue of usage... Thanks for your quick response.
